journal.py

```diff
@@ -112,20 +80,14 @@ def new_entry(request):
     return {}
 
 
-READ_ONE_ENTRY = """
-SELECT id, title, text, created FROM entries WHERE id=%s
-"""
-
-
 def read_one_entry_from_db(request):
     entry_id = request.matchdict.get('id')
-    cursor = request.db.cursor()
-
-    cursor.execute(READ_ONE_ENTRY, [entry_id])
-
-    columns = ('id', 'title', 'text', 'created')
+    entry = Entry.by_id(entry_id)
 
-    return dict(zip(columns, cursor.fetchone()))
+    return {'id': entry.id,
+            'title': entry.title,
+            'text': entry.text,
+            'created': entry.created}
```

Comment:
There's little reason to have a method like this any more.  All this does is convert the Entry object to a dictionary.  You should be able to implement an instance method on Entry that returns relevant data as a dict just as well.